templates/default/setup.ts

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
Tigris SDK will automatically do this for you. Is this for the use case when Tigris SDK package is in a different path in local filesystem?

Reply:
Fixed